### DISCUSSION TITLE
Is redis client's CLUSTER CREATE command equivalent available in redis-py-cluster?

### DISCUSSION THREAD

**Discussion by jackpa99** [Grokzen/redis-py-cluster#495] [Q&A] [answered]
Is CLUSTER CREATE command equivalent available in redis-py-cluster. eg; to do the equivalent of this 
`redis-cli --cluster create 188.175.193.69:3070 188.175.193.69:3073  188.175.193.67:3070 188.175.193.67:3073   --cluster-replicas 1 -a somepassword --cluster-yes`

**Comment by Grokzen** [comment, marked as answer]
@jackpa99 I would say no. Becuase when using this lib you need to have some of the cluster features there and commands enabled etc that only happens after a cluster is created. What i would do instead is to use plain redis-py to connect into your instances and just send `.execute_command("CLUSTER CREATE")` or something similar to each node you want to spin up and manage. But in general it is way simpler to use the redis-cli for this outside any redis-client, that is just from my personal experience.
